x-pack/platform/plugins/shared/cases/server/telemetry/schema.ts

```diff
@@ -68,11 +68,18 @@ const assigneesSchema: AssigneesSchema = {
   totalWithAtLeastOne: long,
 };
 
+const statusSchema: StatusSchema = {
+  open: long,
+  inProgress: long,
+  closed: long,
+};
+
 const solutionTelemetry: SolutionTelemetrySchema = {
   ...countSchema,
   assignees: assigneesSchema,
   attachmentFramework: attachmentFrameworkSchema,
   totalWithAlerts: long,
+  status: statusSchema,
```

Comment:
Given that this was added later and the value does not exist on previous iterations, we might wand to make this `status? : statusSchema`. Wdyt?